runtime/src/main/cpp/Memory.cpp

```diff
@@ -1554,6 +1489,16 @@ void garbageCollect(MemoryState* state, bool force) {
   GC_LOG("<<< GC: toFree %d toRelease %d\n", state->toFree->size(), state->toRelease->size())
 }
 
+void rememberNewContainer(ContainerHeader* container) {
+  if (container == nullptr) return;
+  // Instances can be allocated before actual runtime init - be prepared for that.
+  if (memoryState != nullptr) {
+    IncrementRC</* Atomic = */ false>(container);
```

Comment:
> `/* Atomic = */ false`

Doesn't seem correct when reading from `AtomicReference`.

Reply:
Yeah, noticed that as well.